electroncash/transaction.py

```diff
@@ -458,12 +458,13 @@ def deserialize(self):
         return d
 
     @classmethod
-    def from_io(cls, inputs, outputs, locktime=0, sign_schnorr=False, *, token_datas=None) -> object:
+    def from_io(cls, inputs, outputs, locktime=0, version=1, sign_schnorr=False, *, token_datas=None) -> object:
```

Comment:
Not sure if this will break something somewhere because of the new position of this param -- can you put this after the `*` to make it a kw-only param rather than a positional param?

```suggestion
    def from_io(cls, inputs, outputs, locktime=0, sign_schnorr=False, *, token_datas=None, version=1) -> object:
```